latex.bzl

```diff
@@ -37,19 +37,21 @@ _latex_pdf = rule(
     implementation = _latex_pdf_impl,
 )
 
-def latex_document(name, main, srcs = []):
+def latex_document(name, main, srcs = [], tags = []):
```

Comment:
Just to note: the same change are up for review in #31.

Reply:
Yep, that's because this branch (and thus PR) is based on the commit from #31 

If we land #31 first, then when we update the branch on this PR, we won't see these `tags` changes. Or we could just land this one to include both.